apps/content/api/internal/reciters_crud.py

```
@@ -0,0 +1,101 @@
+    slug: str
+    bio: str
+    is_active: bool
+    created_at: AwareDatetime
+    updated_at: AwareDatetime
+
+
+# ── Endpoints ───────────────────────────────────────────────────────
+
+
+@router.put("reciters/{id}/update/", response=ReciterUpdateOut)
+def update_reciter(request: Request, id: int, data: ReciterUpdateIn):
```

Ruff (0.15.2)

[warning] Unused function argument: `request`

(ARG001)